.github/workflows/actions.yaml

```diff
@@ -20,9 +20,9 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - name: Checkout the code
-        uses: actions/checkout@v2
+        uses: actions/checkout@v4
       - name: Run gosec to check for security vulnerabilities
-        uses: dell/common-github-actions/gosec-runner@main
+        uses: securego/gosec@master
         with:
           directories: "./..."
           excludes: "G108,G402"
```

Comment:
Why are we continuing to exclude things? Remove these please and use //#nosec

Reply:
removed excludes